app/controllers/static_pages_controller.rb

```diff
@@ -2,13 +2,24 @@
 
 # class for static pages
 class StaticPagesController < ApplicationController
+  # root_path  static_pages/home
   def home
     if user_signed_in? && (current_user.user_role.include? 'Employer')
       @jobs = Job.where(user_id: current_user.id)
+      @jobs = Kaminari.paginate_array(@jobs).page(params[:page]).per(7)
       jobs_ids = Job.where(user_id: current_user.id).pluck(:id)
       apply_user_ids = ApplyJob.where('job_id IN (?)', jobs_ids).pluck(:user_id).uniq
       @user_apply = User.where(id: apply_user_ids)
+    end
+
+    if user_signed_in? && (current_user.user_role.include? 'Job Seeker') && (current_user.admin == false)
```

Comment:
Use !current_user.admin


Also update in related code
Check in other files too

Reply:
done